### DISCUSSION TITLE
I have no idea how to set this up, please could you help me owner?

### DISCUSSION THREAD

**Discussion by Avoney** [ImRidingNow/PavlovRconWebserver#88] [General]
Sorry to ask, but I have literally no idea on how to set this up.

Do you have any tutorials anywhere or Discord? Really want to set this up but I have no idea where to start.

Thanks

**Comment by Avoney** [comment]
Looked at the tutorial on the github page but I already have a server, that's the main issue I have and why I'm a bit stuck with this.

**Reply by ImRidingNow** [reply]
Can you tell me a little more? :) Where are you stuck which step or any error? :)

You can add servers to the list even if there are installed manualy :)
